packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -975,6 +975,32 @@ export const components = ({
       html {
         font-size: calc((${themeParam.typography.fontSize} / 16) * 100%);
       }
+
+      :not(code) &, :not(pre) & {
+          :lang(el) {
+            font-family: 'Noto Sans', sans-serif;
+          }
+
+          :lang(ja) {
+            font-family: 'Noto Sans JP', sans-serif;
+          }
+
+          :lang(ko) {
+            font-family: 'Noto Sans KR', sans-serif;
+          }
+
+          :lang(th) {
+            font-family: 'Noto Sans Thai', sans-serif;
+          }
+
+          :lang(zh_CN) {
```

Comment:
I believe this should be `zh-CN`? Since the `lang` element should use dashes

Reply:
see https://developer.mozilla.org/en-US/docs/Web/CSS/:lang

if you replace the `-` with a `_` it will fail